### DISCUSSION TITLE
Close our Twitter account?

### DISCUSSION THREAD

**Discussion by Ainali** [govdirectory/website#422] [General]
I have a bad feeling about that platform and would like to deactivate the account. Does anyone have strong opinions for staying?

**Comment by Abbe98** [comment]
I would like to keep it to avoid loosing the username/someone else taking it over. I would however, have no problem with us pinning a message pointing people elsewhere.

**Reply by Ainali** [reply]
That's a good point. I'll write a tweet that points to Mastodon, probably in some obfuscated form to not get caught in the aggressive filtering of such things and then pin that.

**Reply by Ainali** [reply]
At this point, I care as much if anyone is taking over the username there as if someone registers it on truth.social. What do you think now @Abbe98?

**Reply by Abbe98** [reply]
I think there is a difference between platforms on which we have had an account and ones we have never been on. There are plenty of references to the Twitter account in old presentations videos, etc and we will never get rid of that.
